src/index.js

```diff
@@ -43,6 +43,7 @@ export default class Tooltip {
     this.backgroundColor = config.backgroundColor;
     this.textColor = config.textColor;
     this.editorId = config.holder ? config.holder : 'editorjs';
+    this.inputPlaceholder = config.placeholder ? config.placeholder : "Add a tooltip";
```

Comment:
@abdelhamiderrahmouni I have some suggestions to follow style guides used in the project:

- We could take advantage of the nullish coalescing operator (`??`)
- Consider using single quotes for the default value
- How about renaming `this.inputPlaceholder` to `this.placeholder` for consistency?

```suggestion
    this.placeholder = config.placeholder ?? 'Add a tooltip';
```

Would you mind applying the nullish coalescing operator on lines 42 and 45 as well? 🙌🏻